### PR TITLE
DT-1859 allow using sentry for logging

### PR DESCRIPTION
**Pull request by siren**
In addition to this also mesos-deploy needs update (JAVA_OPTS -Dsentry.dsn=nnn...)

### CODE REVIEW

**Comment by siren** [comment]
WIP

**Comment by siren** [comment]
works in dev

File: src/main/resources/logback.xml [hunk 1]
```diff
@@ -18,9 +18,15 @@
     </encoder>
   </appender>
 
+  <appender name="Sentry" class="com.getsentry.raven.logback.SentryAppender">
+    <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
```
Comment (hannesj) [inline src/main/resources/logback.xml]: Do we want to filter out the warnings/errors from the realtime data sources?

Comment (siren) [inline src/main/resources/logback.xml]: we can only filter on class level, so if set the level higher on a class we miss all other warnings/errors from that class too. 

do we really want to do it?

**Pull request merged**